hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/MoveManager.java

```diff
@@ -264,10 +259,9 @@ CompletableFuture<MoveResult> move(
    */
   private void notifyContainerOpCompleted(ContainerReplicaOp containerReplicaOp,
       ContainerID containerID) {
-    Pair<CompletableFuture<MoveResult>, MoveDataNodePair> pair =
-        pendingMoves.get(containerID);
+    MoveOperation pair = pendingMoves.get(containerID);
```

Comment:
It's not a `pair` anymore, so I suggest renaming the variable to something more suitable.